parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -819,7 +819,7 @@ validateGPUDrivers() {
     fi
 
     retrycmd_if_failure 24 5 25 nvidia-modprobe -u -c0 && echo "gpu driver loaded" || configGPUDrivers || exit $ERR_GPU_DRIVERS_START_FAIL
-    which nvidia-smi
+    command -v nvidia-smi
```

Comment:
it fails in the before?

Reply:
This is a shellcheck fix. Otherwise make generate won't pass. I don't know why it isn't fixed before.